TBAnnotationClustering-Swift/ViewController.swift

```diff
@@ -43,7 +43,7 @@ class ViewController: UIViewController, MKMapViewDelegate {
         let toRemove = NSMutableSet(set: before)
         toRemove.minusSet(after as Set<NSObject>)
         
-        dispatch_async(dispatch_get_main_queue()) { () -> Void in
+        NSOperationQueue().addOperationWithBlock() {
```

Comment:
Good to know :), I'm still learning Swift - that was part of the reason I decided to translate the code - it helped me understand the clustering algorithm better and try out coding in Swift.
Also I find Objective-C's syntax repulsive :P